sql/moz-fx-data-shared-prod/firefox_desktop_derived/ltv_states_v1/query.sql

```diff
@@ -0,0 +1,67 @@
+WITH base AS (
```

Comment:
could this be a view, in principle? we are just transforming existing columns in search_clients_last_seen and doing a couple of joins. i get that the tables involved here are huge though, so view performance may not be great. I'm far from an expert on these things so just throwing it out there as a possibility

Reply:
It could definitely also be a view, I just thought making it a table would make it more performant / faster for the following steps that run off of it (i.e. the client_ltv_v1 table).  It's not a bad idea though, we can always try switching it to view later and see if it does ok